indigo_api/templates/indigo_api/akn/export/standalone.html

```diff
@@ -23,7 +23,7 @@ <h3>{{ document.title }}</h3>
     </div>
   {% endif %}
 
-  <div class="akoma-ntoso country-{{ document.country }}">
+  <la-akoma-ntoso frbr-expression-uri="{{ document.expression_frbr_uri }}">
```

Comment:
This can also remain as-is. We use it when generating a version of HTML that has the CSS embedded into it, using the export CSS file.

I suspect later we can use la-akoma-ntoso and still apply styles to it, even though it won't have corresponding JS functionality. But we can cross that bridge later.

Reply:
reverted